packages/frontend/src/pages/UserActivity.styles.ts

```diff
@@ -20,7 +19,11 @@ export const Container = styled.div`
         'table-dashboard-views table-dashboard-views table-chart-views table-chart-views';
 `;
 
-export const ActivityCard = styled(Card)<{ grid: string }>`
+export const ActivityCard = styled.div<{ grid: string }>`
+    background-color: #fff;
+    border-radius: 2px;
+    padding: 20px;
+    border: 1px solid #dee2e6;
```

Comment:
Styles that used to come from the card

Reply:
we can use mantine's Card component here. but if its too much work then ignore